docs/property_handling.md

```diff
@@ -0,0 +1,28 @@
+# Property Handling
+
```

Comment:
Should we add something to this section about how the playbooks can reference properties, including system and deployment location properties? And show examples of how they can do this?